cassandra-analytics-common/src/main/java/org/apache/cassandra/spark/utils/Properties.java

```diff
@@ -40,6 +40,8 @@ public final class Properties
     public static final int DEFAULT_MAX_RETRIES = 10;
     public static final long DEFAULT_MILLIS_TO_SLEEP = 500;
     public static final int DEFAULT_MAX_POOL_SIZE = 64;
+    public static final boolean DEFAULT_CACHE_COMPRESSION_METADATA_KEY = true;
+    public static final int DEFAULT_MAX_SIZE_CACHE_COMPRESSION_METADATA_KEY = 8388608; // 8MiB
```

Comment:
```suggestion
    public static final int DEFAULT_MAX_SIZE_CACHE_COMPRESSION_METADATA_KEY = 8 * MEBI_BYTES; // 8MiB
```

Reply:
for future reference, shifting left in multiples of 10 multiplies by KiB. so 8 << 20 is a clean way to say 8 Mebibytes